material-stepper/src/main/java/com/stepstone/stepper/StepperLayout.java

```diff
@@ -496,6 +509,11 @@ private void onNext() {
         if (verifyCurrentStep(step)) {
             return;
         }
+
+        //if moving forward and got no errors, set hasError to false, so we can have the tab with the check mark.
+        if(mStepperType instanceof TabsStepperType)
```

Comment:
Hi,
I'm considering adding the alternative tabs look so it might be a pain in the a*s to update all those ifs :/
Could you replaced them with the proposed solution so that it's easier to maintain it later?